projects/plugins/jetpack/extensions/blocks/videopress/edit.js

```diff
@@ -192,6 +196,35 @@ const VideoPressEdit = CoreVideoEdit =>
 			return media;
 		};
 
+		setTracks = async guid => {
+			const { setAttributes } = this.props;
+
+			if ( ! guid ) {
+				return;
+			}
+
+			await apiFetch( {
+				url: `https://public-api.wordpress.com/rest/v1.1/videos/${ guid }`,
+				credentials: 'omit',
+				global: true,
+			} ).then( videoInfo => {
```

Comment:
if we use async/await then the `then()` part is not needed, we can instead do `const videoInfo = await apiFetch(...);`

On the other hand, since this code seems pretty Promise-based, we can just go all in and: 1. get rid of await 2. add catch/finally to the promise.

Reply:
Updated in bb30f90! I don't think we need to take any action in a `catch` or `finally` in this case. If we don't get the video info back, then we simply won't set the tracks data.